src/screens/AddNewItemScreen/index.tsx

```diff
@@ -82,6 +105,39 @@ export const AddNewItemScreen: FC = () => {
     setOpen(true);
   };
 
+  if (isEdit) {
+    return (
+      <View style={style.container}>
+        <View style={style.inputContainer}>
+          <CustomInput
+            onChangeText={setText}
+            value={text}
+            title={'Edit tack'}
+          />
+          <CustomInput
+            value={currentDate}
+            title={'Edit the reminder send time'}
+            onPress={openDate}
+          />
+        </View>
+        <DatePicker
+          minimumDate={new Date()}
+          modal
+          open={open}
+          date={date}
+          onConfirm={onConfirmDate}
+          onCancel={onCancelDate}
+        />
+        <TouchableOpacity
+          disabled={!text}
+          style={buttonStyle}
+          onPress={onPress}>
+          <Text style={style.text}>{'Edit task'}</Text>
+        </TouchableOpacity>
+      </View>
```

Comment:
why do you duplicate this code that you already have for adding functionality?

Reply:
i fix it